core/src/main/java/nl/nn/adapterframework/filesystem/IBasicFileSystem.java

```diff
@@ -47,21 +47,21 @@
 
 	public boolean isOpen();
 
-
 	/**
-	 * Lists all files in 'folder' or in the 'root' of the filesystem (when folder is null). 
+	 * Lists all files in 'folder' or in the 'root' of the filesystem (when folder is null).
 	 * Should list only 'files', no folders.
 	 */
-	public DirectoryStream<F> listFiles(String folder) throws FileSystemException;
+	public DirectoryStream<F> listFiles(String folder, boolean ignoreFolders) throws FileSystemException;
```

Comment:
There should not be a new parameter `ignoreFolders`, listFiles should never return folders.